cmake/Modules/Findlibusb.cmake

```diff
@@ -66,7 +66,7 @@ find_package_handle_standard_args(libusb DEFAULT_MSG libusb_LIBRARIES libusb_INC
 
 mark_as_advanced(libusb_INCLUDE_DIRS libusb_LIBRARIES)
 
-if(LIBUSB_FOUND)
+if(libusb_FOUND)
```

Comment:
Is it right that libusb is lowercase here, but uppercase (`LIBUSB_FOUND`) everywhere else? Would it maybe be a good idea to also set `LIBUSB_FOUND` if `libusb_FOUND` is true?

Reply:
CMake variables are not case sensitive, so both are correct. Since `libusb` itself writes its name in lowercase, we use `find_package(libusb)` (there the case is important, because CMake then searches for `Findlibusb.cmake`). Usually CMake then also defines the variables in the same notation, i.e. `libusb_FOUND`. Stupidly vcpkg doesn`t stick to this, but makes it as `LIBUSB_FOUND`.

Reply:
Cmake variables are case sensitive? https://cmake.org/cmake/help/latest/manual/cmake-language.7.html#variables

The find_package_handle_standard_args sets the inputted name as it is, when defining xxx_FOUND. So since its suppyling libusb, it should be libusb_FOUND.

Maybe the variables in pcl_find_libusb could be changed to lower case for consistency. I'll investigate.

Reply:
I have set it to lowercase. However, I read up on find_package_handle_standard_args and it sets both the lowercase and uppercase _FOUND variable for compatibility.

Reply:
Ah, that explains why I thought it didn't matter whether case sensitive or not. Wrote to less CMake code in the last month^^